core/input/input_event.h

```diff
@@ -79,7 +79,7 @@ class InputEvent : public Resource {
 
 	virtual Ref<InputEvent> xformed_by(const Transform2D &p_xform, const Vector2 &p_local_ofs = Vector2()) const;
 
-	virtual bool action_match(const Ref<InputEvent> &p_event, bool *p_pressed, float *p_strength, float *p_raw_strength, float p_deadzone) const;
+	virtual bool action_match(const Ref<InputEvent> &p_event, const bool p_exact_match, const float p_deadzone, bool *const p_pressed, float *const p_strength, float *const p_raw_strength) const;
```

Comment:
All the added const qualifiers are not useful here, and make the signature harder to read and inconsistent with the rest of the codebase.